apps/package.json

```diff
@@ -101,6 +101,7 @@
     "react-redux": "^4.4.0",
     "redux": "^3.3.1",
     "redux-logger": "^2.6.1",
+    "redux-thunk": "^2.0.1",
```

Comment:
[`redux-thunk`](https://github.com/gaearon/redux-thunk) allows your action-creators to return functions instead of POJOs.  Those functions in turn are able to read state and dispatch additional actions.  This provides a neat template for dispatching asynchronous actions, and is [a recommended way to handle async work in redux](http://redux.js.org/docs/advanced/AsyncActions.html).